Apps/W1/APIV2/test/src/APIV2FALocationsE2E.Codeunit.al

```diff
@@ -0,0 +1,161 @@
+codeunit 139900 "APIV2 - FA Locations E2E"
+{
+    // version Test,W1,All
+
+    Subtype = Test;
+    TestPermissions = Disabled;
+
+    trigger OnRun()
+    begin
+        // [FEATURE] [Api] [Location]
+    end;
+
+    var
+        Assert: Codeunit Assert;
```

Comment:
Maybe it would be better to use "Library Assert" in new test codeunits? It's a new version of the Assert codeunit, but better structured and documented, and generally recommended for new development.

Reply:
I was following the example set in other API V2 test codeunits, but maybe it will be truly better to use the "Library Assert". There is only a question about the consistency across all test codeunits. From what I saw most of them (tests for API V2) use Assert codeunit. What are your thoughts on this matter @JesperSchulz?
![image](https://github.com/microsoft/ALAppExtensions/assets/114576403/2e806cc9-2ecf-46c3-a470-154f5dd87992)

Reply:
Normally it is indeed better to use the Library, but I too prefer consistency. Makes refactoring easier, when the day comes, while keeping a recognizable pattern until that day. I agree with this solution.